pyglet/math.py

```diff
@@ -130,12 +156,45 @@ def __truediv__(self, other):
     def __abs__(self):
         return _math.sqrt(self[0] ** 2 + self[1] ** 2 + self[2] ** 2)
 
+    def cross_product(self, other):
```

Comment:
Could also be named `cross`

Reply:
I'm in favor of that.  We already just have other things like "dot", instead of "dot_product", so I think it makes sense to keep them consistent.